integrations/zendesk/src/definitions/actions.ts

```diff
@@ -181,6 +181,44 @@ const openTicket = {
   },
 } satisfies NonNullable<IntegrationDefinitionProps['actions']>[string]
 
+const makeApiRequest = {
+  title: 'Make API Request',
+  description: 'Make request to Zendesk API',
+  input: {
+    schema: z.object({
+      method: z.string().describe('HTTP Method'),
+      path: z.string().describe('URL Path (https://<subdomain>.zendesk.com/api/PATH)'),
+      headers: z.string().optional().describe('Headers (JSON)'),
+      params: z.string().optional().describe('Query Params (JSON)'),
+      requestBody: z.string().optional().describe('Request Body (JSON)'),
+    }),
+  },
+  ui: {
+    method: {
+      title: 'HTTP Method',
+    },
+    path: {
+      title: 'URL Path (https://<subdomain>.zendesk.com/api/PATH)',
+    },
+    headers: {
+      title: 'Headers (JSON)',
+    },
+    params: {
+      title: 'Params (JSON)',
+    },
+    requestBody: {
+      title: 'Request Body (JSON)',
+    },
+  },
+  output: {
+    schema: z.object({
```

Comment:
**chore:** we should return a response schema here:
```ts
z.object({
  status: z.number(),
  headers: z.record(z.string(), z.string()),
  body: z.any()
})
```

so the action user can interpret the response as he wants